sections/appendix/problems.tex

```diff
@@ -49,14 +49,13 @@ \subsubsection{Unable to revert the \gls{lda} generative process}
 To test this initial query handling we generated some queries by taking the 4 highest-ranking words in the document using tf-idf. Using this query we would then see if our algorithm could find the document again only using the words in the query.
 
 We hoped that each word would only be a significant part of a few topics and that each topic was only a significant part of a few documents.
-If the topic-distribution only contained a few high values, one or more of which would be among the topics belonging to the original document (which is a stand-in for a relevant document in this case).
-The original document would then hopefully be a significant part of these topics, leading us back to the document.
+If the topic-distribution only contained a few high values, one or more of which would be among the topics belonging to the original document, then we might be able to find the original document, based on the query.
 
 This reverse engineering process formed the base of our query handling, where we converted a query of words into a personalization vector (containing a value between 0 and 1 for each document).
 
-However, when we tested this process, we found that the distributions were far too evenly distributed between their values to get much meaning out of them.
+However, when we tested this process, we found that the distributions were far too evenly valued to get much meaning out of them.
 To remedy this, we made the distributions contain fewer values by converting all values under a given threshold to 0 and then normalizing the values.
 While this resulted in more decisive distributions, the results were very bad, often ranking the original document very low.
-Furthermore significantly under half the words would even have a single matching topic with the original document, despite usually having more topics in their distribution than the document's distribution.
+Furthermore, significantly under half the words from the query would have even a single matching topic with the original document, despite usually having more topics in their distribution than the document's distribution.
```

Comment:
Jeg tænker "significantly" er et lidt for kraftigt ord at bruge i denne sammenhæng, da jeg sidst hørte at vi testede på queries med 4 ord.

Reply:
denne test var faktisk på queries med 10-20 ord, fordi at jeg ville være sikker på at det ikke var for lidt context som gjorde algoritmen dårlig